integration/pom.xml

```diff
@@ -77,18 +76,6 @@
       <scope>test</scope>
     </dependency>
 
-    <dependency>
-      <groupId>org.eclipse.jetty</groupId>
-      <artifactId>test-jetty-servlet</artifactId>
-      <version>${test-jetty-servlet.version}</version>
-      <scope>test</scope>
```

Comment:
Here you are actually changing the scope. If the scope is test here, it should be maintained as scope.